src/components/GigBoard/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import useServices from '../../hooks/useServices';
+import { ServiceStatusEnum } from '../../types';
+import ServiceItem from '../ServiceItem';
+
+interface IGigBoardProps {
```

Comment:
The semantic used for the end user and the one use in the code is different. In the code, we use Service and Gig.